src/index.js

```diff
@@ -45,7 +46,8 @@ const trackNavigation = (location) => {
   const locationPage = location.hash && location.hash[0] === '#' && location.hash.slice(1);
   ReactGA.set({ page: locationPage });
   ReactGA.pageview(locationPage);
-  window._dha && window._dha.track({ navigatedPage: locationPage });
+  const dataHubParams = getDataHubParams(locationPage);
+  window._dha && window._dha.track({ ...dataHubParams });
```

Comment:
@cbernat, I am ingnorant in the use of  `...` (among other things :D ). But would not be enough call to `track` in this way: `_dha.track(dataHubParams)`. 
What extra value does the 3 points add?

Reply:
I agree, as we talked earlier, I will change this. Thanks for the insight Leo!

Reply:
good catch!

if I am not wrong, both these calls are aquivalent:

```js
f(obj);
f({...obj});
```

Well, in fact, there is a minor difference. But I think that it does not affects us:

Imagine this:

```js
function f(obj) {
  obj.anotherProp = 'added';
}

const t1 = { test: 't1' };
f(t1);
console.log(JSON.stringify(t1)); 
// { "test": "t1", "anotherProp": "added" }

const t2 = { test: 't2' };
f({ ...t2 });
console.log(JSON.stringify(t2)); 
// { "test": "t2" }
```